backend/__tests__/userDeleteBooking.test.js

```
@@ -0,0 +1,331 @@
+            `${user.username} has canceled their booking for your property from ${booking.checkIn.toDateString()} to ${booking.checkOut.toDateString()}.`,
+            'cancellation'
+          );
+
+          return res.status(200).json({ success: true, message: 'Booking canceled successfully.' });
+        } catch (err) {
+          console.error("Delete Booking Error:", err);
+          return res.status(500).json({ message: 'Server error while canceling booking.' });
+        }
+      } catch (error) {
+        // Simply rethrow the error for testing error handling
+        throw error;
```

Biome (1.9.4)

[error] The catch clause that only rethrows the original error is useless.

An unnecessary catch clause can be confusing.
Unsafe fix: Remove the try/catch clause.


(lint/complexity/noUselessCatch)